lesson/day01.md

```diff
@@ -5,103 +5,109 @@ title: Day 1
 subtitle: Introduction
 ---
 
-## Requirements for Learning
+# Prerequisites
+  ---------------------
 
-To learn assembly programming for the TI-83 Plus, you will require a few
-things. This lesson will guide you through setting up all the required tools
-and creating your first program.
+In order to use this guide, you must have some experience with C or a similar programming 
```

Comment:
I don't think this is true; the original notes this is "recommended" and it's definitely not mandatory to have knowledge of C. I think a lot of readers come from TI-BASIC and find it intelligible enough.

Reply:
My apologies. I let my personal bias get in the way. C is now recommended.